src/lib/eina/eina_lock.h

```diff
@@ -19,6 +19,12 @@
 #ifndef EINA_LOCK_H_
 #define EINA_LOCK_H_
 
+#ifdef _WIN32
+# ifndef WIN32_LEAN_AND_MEAN
```

Comment:
why not just in eina_lock_win32.c ?

Reply:
I don't remember actually... I will remove it.

Reply:
Oh, just remembered. As this file include others, there is a chance that they include windows things, so it is there for precaution. But maybe it is removable, what do you guys think, @JPTIZ and @caiosignor ?

Reply:
Ok, it make sense